docs/source/using-executorch-export.md

```diff
@@ -1,3 +1,173 @@
-# Model Export
+# Model Export and Lowering
 
-Placeholder for top-level export documentation
\ No newline at end of file
+The section describes the process of taking a PyTorch model and converting to the runtime format used by ExecuTorch. This process is commonly known as "exporting", as it uses the PyTorch export functionality to convert a PyTorch model into a format suitable for on-device execution. This process yields a .pte file which is optimized for on-device execution using a particular backend.
+
+## Prerequisites
+
+Exporting requires the ExecuTorch python libraries to be installed, typically by running `pip install executorch`. See [Installation](getting-started.md#Installation) for more information. This process assumes you have a PyTorch model, can instantiate it from Python, and can provide example input tensors to run the model.
+
+## The Export and Lowering Process
+
+The process to export and lower a model to the .pte format typically involves the following steps:
+
+1) Select a backend to target.
+2) Prepare the PyTorch model, including inputs and shape specification.
+3) Export the model using torch.export.export.
+4) Optimize the model for the target backend using to_edge_transform_and_lower.
+5) Create the .pte file by calling to_executorch and serializing the output.
+
```

Comment:
Add a disclaimer that quantization is a separate optional step and this page doesn't talk.